tests/integration/ambient/baseline_test.go

```diff
@@ -1143,6 +1143,93 @@ spec:
 	})
 }
 
+func TestAuthorizationWaypointDefaultDeny(t *testing.T) {
+	framework.NewTest(t).Run(func(t framework.TestContext) {
+		applyDrainingWorkaround(t)
+		runTestContext(t, func(t framework.TestContext, src echo.Instance, dst echo.Instance, opt echo.CallOptions) {
+			if !dst.Config().HasServiceAddressedWaypointProxy() {
```

Comment:
Workload address waypoints don't have default deny policy applied?

Reply:
Ack, not necessary here; I do see some workload targeting further down I think; is it ineffectual because of this early return?

Reply:
Ack, not necessary here; I do see some workload targeting further down I think; is it ineffectual because of this early return?

Reply:
oh, good call out. I did start down the path of making that work and then decided against it but didn't fully clean it up. The issue isn't setting up the policy, that wasn't so bad. The issue is that if src calls a service the wl-attached waypoint isn't used so you need to add logic to change the test opts such that the call will address some workload instead of the service.

it exists in other tests. lemme see if I can just copy that logic over and enable full waypoint coverage afterall. otherwise I can back out this unneeded logic for getting the waypoint name

Reply:
I took another look and I think it's not so hard to send traffic to the workload instead of the service